src/ast/passes/codegen_llvm.cpp

```diff
@@ -1490,11 +1490,14 @@ void CodegenLLVM::visit(Call &call)
     b_.CreateStore(b_.CreateGetNs(TimestampMode::boot, call.loc), time_addr);
 
     auto &arg_skb = *call.vargs.at(1);
-    arg_skb.accept(*this);
+    // FIXME(amscanne): this is a likely bug, as we are note appropriately
```

Comment:
```suggestion
    // FIXME(amscanne): this is likely a bug, as we are not appropriately
```

Reply:
You could also create an issue for these two lifetime bugs and add it to the comment.